ui/src/components/Settings/About/index.tsx

```diff
@@ -0,0 +1,322 @@
+        <div className="flex">
+          <div className="flex-shrink-0">
+            <InformationCircleIcon className="h-5 w-5 text-gray-100" />
+          </div>
+          <div className="ml-3 flex-1 md:flex md:justify-between">
+            <p className="text-sm leading-5 text-gray-100">
+              This is BETA software. Features may be broken and/or unstable.
+              Please report any issues on GitHub!
+            </p>
+            <p className="mt-3 text-sm leading-5 md:ml-6 md:mt-0">
+              <a
+                href="http://github.com/jorenn92/maintainerr"
+                className="whitespace-nowrap font-medium text-gray-100 transition duration-150 ease-in-out hover:text-white"
+                target="_blank"
+                rel="noreferrer"
+              >
+                GitHub &rarr;
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+      {/* Maintainerr Portion */}
+      <div className="section mb-2 h-full w-full">
+        <h3 className="heading">About Maintainerr</h3>
+      </div>
+      <hr className="my-2 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+      <div className="section my-2">
+        <div className="form-row my-2">
+          <label htmlFor="name" className="text-label">
+            Version
+          </label>
+          <div className="form-input">
+            <div className="form-input-field">
+              <span className="">
+                <code>{commitTag === 'local' ? 'local' : <>{version}</>}</code>
+              </span>
+            </div>
+          </div>
+        </div>
+        <hr className="my-2 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+        <div className="form-row my-2">
+          <label htmlFor="name" className="text-label">
+            Container Config Path
+          </label>
+          <div className="form-input">
+            <div className="form-input-field">
+              <span className="">
+                <code>/opt/data</code>
+              </span>
+            </div>
+          </div>
+        </div>
+        <hr className="my-2 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+        <div className="form-row my-2">
+          <label htmlFor="name" className="text-label">
+            Time Zone
+          </label>
+          <div className="form-input">
+            <div className="form-input-field">
+              <span className="">
+                <code>{getBrowserTimezone()}</code>
+              </span>
+            </div>
+          </div>
+        </div>
+        <hr className="my-2 h-px border-0 bg-gray-200 dark:bg-gray-700"></hr>
+        <div className="form-row my-2">
+          <label htmlFor="name" className="text-label">
+            Number Of Rules
+          </label>
+          <div className="form-input">
+            <div className="form-input-field">
+              <span className="">
+                <code>{<>{ruleCount}</>}</code>
```

Comment:
These can all be simplified to:

```suggestion
                <code>{ruleCount}</code>
```